ui/src/pages/AllStories.tsx

```diff
@@ -0,0 +1,210 @@
+import Checkbox from "@material-ui/core/Checkbox";
+import Switch from "@material-ui/core/Switch";
+import { useState } from "react";
+import * as React from "react";
+import useSWR from "swr";
+
+import VirtualizedTable from "../components/VirtualizedTable";
+import { Story } from "../types/index";
+
+function createData(
+  id: number,
+  title: string,
+  current_city: string,
+  year: number,
+  is_visible: boolean,
+  author_first_name: string,
+  author_last_name: string,
+  author_country: string
+) {
+  const author_name = author_first_name + " " + author_last_name;
+  return {
+    id,
+    title,
+    current_city,
+    year,
+    author_name,
+    author_country,
+    is_visible,
+  };
+}
+
+export const AllStories: React.FC = () => {
+  const { data: allStories, error } = useSWR<Story[]>("/api/stories");
+  let rows = [];
+  if (allStories) {
+    rows = allStories.map((story, i) =>
+      createData(
+        i,
+        story.title,
+        story.current_city,
+        story.year,
+        story.is_visible,
+        story.author_first_name,
+        story.author_last_name,
+        story.author_country
+      )
+    );
+  }
+
+  const [visibleState, setVisibleState] = React.useState([]);
+  const [tableData, setTableData] = useState(rows);
+  const [selectedRowIds, setSelectedRowIds] = useState([]);
+  const [order, setOrder] = useState("desc");
+  const [orderBy, setOrderBy] = useState("age");
+
+  const handleChange = (e, d) => {
+    if (e.target.checked) {
```

Comment:
We know what story the checkbox/toggle state belongs to based off of ID. We can also create a tabledata type but there isn't one currently.